features/step_definitions/seeding_steps.rb

```diff
@@ -8,3 +8,8 @@
   SHFProject::Application.load_tasks
   SHFProject::Application.load_seed
 end
+
+Then(/^(\d+) "([^"]*)" records should be created$/) do |number_of_records, models|
+  model_klass = ActiveSupport::Inflector.singularize(models)
+  expect((eval "#{model_klass}").all.size).to eq(number_of_records.to_i)
```

Comment:
This construct was already used in the seeding_steps.rb file, so I take it the security risk has been flagged and discussed earlier?